stellar/deploy-contract.js

```diff
@@ -91,6 +102,15 @@ async function deploy(options, config, chain, contractName) {
     );
     const wasmHash = await uploadWasm(wasmPath, wallet, chain);
 
+    if (contractName === 'interchain_token') {
+        chain.contracts[contractName] = {
+            deployer: wallet.publicKey(),
+            wasmHash: serializeValue(wasmHash),
```

Comment:
store deployer and wasmHash for all contracts, doesn't hurt to have it

Reply:
agreed. updated